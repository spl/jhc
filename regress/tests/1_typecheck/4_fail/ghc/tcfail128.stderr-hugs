ERROR "tcfail128.hs":12 - Cannot justify constraints in explicitly typed binding
*** Expression    : main
*** Type          : IO ()
*** Given context : ()
*** Constraints   : MArray a (UArray Int Double) IO
