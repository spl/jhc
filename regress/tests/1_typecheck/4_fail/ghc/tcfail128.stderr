
tcfail128.hs:18:16:
    No instance for (Data.Array.Base.MArray b0 FlatVector IO)
      arising from a use of `thaw'
    Possible fix:
      add an instance declaration for
      (Data.Array.Base.MArray b0 FlatVector IO)
    In a stmt of a 'do' block: v <- thaw tmp
    In the expression:
      do { let sL = ...
               dim = length sL
               ....;
           v <- thaw tmp;
           return () }
    In an equation for `main':
        main
          = do { let sL = ...
                     ....;
                 v <- thaw tmp;
                 return () }
